docarray/typing/tensor/video/abstract_video_tensor.py

```diff
@@ -0,0 +1,99 @@
+from abc import ABC
+from typing import BinaryIO, Optional, TypeVar, Union
+
+import numpy as np
+
+from docarray.typing.tensor.abstract_tensor import AbstractTensor
+from docarray.typing.tensor.audio.audio_tensor import AudioTensor
+
+T = TypeVar('T', bound='AbstractVideoTensor')
+
+
+class AbstractVideoTensor(AbstractTensor, ABC):
+    def save_to_mp4_file(
```

Comment:
I would rather just call it `save()`, and in the future potentially allow the format as an argument. Otherwise we might again end up in the situation where we have `save_to_x_file()` for 7 different x

Reply:
agree